test/specs/behaviors/testHelper.tsx

```diff
@@ -60,25 +67,39 @@ export class TestHelper {
   public findRegexAndAssingCorrespondingInfoToArray(behaviorMenuItems: any) {
     behaviorMenuItems.forEach(behavior => {
       behavior.variations.forEach(variant => {
-        variant.text.split('\n').forEach(singleLineText => {
-          this.iterateRegexDefinitions(singleLineText, variant.name)
-        })
+        if (!variant.specification && !variant.text) {
```

Comment:
it is very hard to reason what `variation.text` stands for. Lets rename it to `variation.description`, as this, in fact, what it is